com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/PathTracing/Shaders/PathTracingMain.raytrace

```diff
@@ -53,6 +57,13 @@ void MissMaterial(inout PathIntersection pathIntersection : SV_RayPayload)
     ApplyFogAttenuation(WorldRayOrigin(), WorldRayDirection(), pathIntersection.value);
 }
 
```

Comment:
I would go with:
float2 SampleDiskUniform(float u1, float u2)
{
    float r = sqrt(u1);
    float theta = u2 * TWO_PI;
    return float2(r * cos(theta), r * sin(theta));
}
and then multiply the radius from the caller. Also, move this in PathTracingSampling.hlsl.
We should check whether this was not implemented in HDRP already, there are some sampling routines around.

Reply:
See above, it is already in Sampling.hlsl